.github/workflows/ci.yml

```diff
@@ -29,6 +29,6 @@ jobs:
                     tools: composer:v2
                     coverage: none
             -   name: Install PHP dependencies
-                run: composer update ${{ matrix.dependency-version }} --prefer-dist --no-interaction --no-progress --no-suggest --ignore-platform-reqs
+                run: composer update ${{ matrix.dependency-version }} --prefer-dist --no-interaction --no-progress --ignore-platform-req=ext-pdo
```

Comment:
Don’t ignore the PHP version when installing dependencies to prevent doctrine/dbal version being installed that is incompatible with PHP 8.0. Only ignore the PDO requirement.

Also, `--no-suggest` is deprecated, so remove it.